src/python/scripts/run_gerp.py

```diff
@@ -28,45 +28,78 @@
     $ python run_gerp.py --msa_file alignment.mfa --tree_file tree.nw --gerp_exe_dir path/to/gerp/
 
 """
-
+import argparse
+import json
 import os
 import subprocess
 
-import argschema
+def add_ce_to_json(ce_file: str, json_file: str) -> None:
+    """Enrich the json file with the constrained elements.
 
+    Args:
+        ce_file: file containing the constrained elements.
```

Comment:
It would be helpful to specify the type/format of the `ce_file`, for example, is it a `tsv` file? A `csv` file?